configs/machines/all_machines.yaml

```
@@ -54,3 +54,8 @@ nesh:
                - 'nesh-bigmem*'
         post_nodes: 'neshcl*'
 
+ecmwf-atos:
+        login_nodes: 'ac*-*'
+        compute_nodes: 'ac*-*'
+        post_nodes: 'ac*-*'
```

@chrisdane, @tsemmler05, I found this string in the `env.sh` of one of the latest branches of FESOM2, probably written by Thomas R., so my guess is that this could work.
```suggestion
        login_nodes: '\.bullx$'
        compute_nodes: '\.bullx$'
        post_nodes: '\.bullx$'
```

---

This doesn't seem to work. I get 

WARNING:root:The yaml file for this computer (ac6-100.bullx) could not be determined!
WARNING:root:Continuing with generic settings...

---

Okay, I'll look into this when I get access to the system

---

probably just wild card missing? `*\.bullx$`